spatialconnect/src/main/java/com/boundlessgeo/spatialconnect/services/SCBackendService.java

```diff
@@ -299,54 +301,16 @@ private void loadCachedConfig () {
     }
 
     private void authListener() {
```

Comment:
The authListener is in parity with iOS, I don't think we need to change anything here.  Looking at iOS no auth PR the authListener didn't change.